interface/views.py

```diff
@@ -296,3 +296,16 @@ def subs_for_user(request, course_pk, assignment_pk, username):
         'assignment': assignment,
         'submissions': submissions,
     })
+
+
+@login_required
+def user_page(request, username):
+    user = get_object_or_404(User, username=username)
+    submissions_aux = Submission.objects.all().order_by('-timestamp')
+    submissions = submissions_aux.filter(user=user)
```

Comment:
This can be a single line. (e.g: `Submission.objects.all().filter(user=user).order_by('-timestamp')`)